easybuild/easyconfigs/p/Pillow/Pillow-5.3.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '5.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2ea3517cd5779843de8a759c2349a3cd8d3893e03ab47053b66d5ec6f8bc4f93']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('libjpeg-turbo', '2.0.0'),
+    ('libpng', '1.6.34'),
+    ('zlib', '1.2.11'),
+    ('LibTIFF', '4.0.9'),
+    ('freetype', '2.9.1')
+]
+
+download_dep_fail = True
+use_pip = True
+
+options = {'modulename': 'PIL'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```

Comment:
This is the default in PythonPackage, you can skip sanity_check_paths

Reply:
True, fixed :+1: